qiskit_nature/problems/second_quantization/electronic/electronic_structure_problem.py

```diff
@@ -175,6 +175,48 @@ def get_default_filter_criterion(
 
         In the fermionic case the default filter ensures that the number of particles is being
         preserved.
+
+        .. note::
+
+            The default filter_criterion assumes a singlet spin configuration. (the number of alpha
+            particles is equal to the number of beta particles).
+            If the AngularMomentum property is available, then working with non singlet spin
+            configuration can be done using a custom filter_criterion in the following manner:
```

Comment:
```suggestion
            If the AngularMomentum property is available, one can correctly filter a non-singlet spin
            configuration with a custom `filter_criterion` similar to the following:
```
Please also add a link to the `AngularMomentum` class (see some other docstrings which use the `:class:` syntax on how to do this)